tools/tools_common.h

```diff
@@ -115,11 +115,11 @@ disableLogging(dht::DhtRunner& dht)
 /**
  * Converts std::chrono::duration to floating-point seconds.
  */
-template <class DT>
+/*template <class DT>
```

Comment:
Shouldn't we remove function that are not used ? 
Since we've got a version tracker thanks  to git.